server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestUpload.java

```diff
@@ -173,13 +174,11 @@ private void handleUpload(final Request the_request,
             if (total >= MAX_UPLOAD_SIZE) {
               Main.LOGGER.info("attempt to upload file greater than max size from " + 
                                raw.getRemoteHost());
-              the_info.my_response_string = "Upload Failed";
-              the_info.my_response_status = HttpStatus.UNPROCESSABLE_ENTITY_422;
```

Comment:
This was something you needed to look at further because of the way I was tracking statuses on the upload endpoints. So I looked at it further.